src/runtime_src/core/tools/common/ReportAie.cpp

```diff
@@ -23,6 +23,10 @@
 #include <boost/property_tree/json_parser.hpp>
 
 #define fmtCommon(x) boost::format("    %-22s: " x "\n")
+#define fmt4(x) boost::format("    %-22s: " x "\n")
+#define fmt8(x) boost::format("        %-22s: " x "\n")
+#define fmt12(x) boost::format("            %-22s: " x "\n")
+#define fmt16(x) boost::format("                %-22s: " x "\n")

```

Comment:
Argh, #define.  Can these be inline functions?  Not a very important comment, feel free to ignore

Reply:
Please do refer to the boost::format documentation.  What you are desiring to do it can already do so e.g., programmatically inserting spaces.